libs/islamic-website/index-page/src/lib/funds-block/funds-block.tsx

```diff
@@ -51,7 +51,12 @@ export function FundsBlock() {
               aria-label="Optic Capital"
             >
               <div className="relative h-[80px] w-[100px] opacity-60 transition-opacity duration-300 hover:opacity-100">
-                <Image src={opticCapitalLogoImgData.src} alt="" fill />
+                <Image
+                  src={opticCapitalLogoImgData.src}
```

Comment:
better to use image data as src prop for next images
```suggestion
                  src={opticCapitalLogoImgData}
```

Reply:
Done